vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -154,7 +154,8 @@ NativeUIManager::NativeUIManager(winrt::Microsoft::ReactNative::ReactContext con
   if (React::implementation::QuirkSettings::GetUseWebFlexBasisBehavior(m_context.Properties()))
     YGConfigSetExperimentalFeatureEnabled(m_yogaConfig, YGExperimentalFeatureWebFlexBasis, true);
   if (React::implementation::QuirkSettings::GetMatchAndroidAndIOSStretchBehavior(m_context.Properties()))
-    YGConfigSetUseLegacyStretchBehaviour(m_yogaConfig, true);
+    YGConfigSetErrata(m_yogaConfig, YGErrataStretchFlexBasis);
+    // YGConfigSetUseLegacyStretchBehaviour(m_yogaConfig, true);
```

Comment:
Just confirming this function commented out is intentional.

Reply:
good catch! Must of left that in accidentally when fixing the build